cmd/integration/integration.go

```diff
@@ -93,17 +94,24 @@ func (h *delegateHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 func startComponents(firstManifestURL, secondManifestURL string) (string, string) {
 	// Setup
-	servers := []string{}
-	glog.Infof("Creating etcd client pointing to %v", servers)
-
 	handler := delegateHandler{}
 	apiServer := httptest.NewServer(&handler)
 
-	etcdClient := etcd.NewClient(servers)
+	cfg := etcd.Config{
+		Endpoints: []string{"http://127.0.0.1:4001"},
+		Transport: etcd.DefaultTransport,
```

Comment:
not super familiar with the new etcd api -- why is `Transport` assigned to `etcd.DefaultTransport` for some of these configs and not for others?